packages/ia-user-list/package.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "@internetarchive/ia-userlist-settings",
```

Comment:
@bfalling noted we should prefix our components with `<iaux-foo-bar>` please update to `iaux-userlist-settings` for component name

Reply:
Currently what I can see on our different repo/component. the repo starts with `/iaux-userlist-settings` but and what we publish is `ia-userlist-settings`. So for now, I am just making consistency here.